src/i_oalsound.c

```diff
@@ -39,10 +39,10 @@
 #include "w_wad.h"
 #include "z_zone.h"
 
-#define OAL_ROLLOFF_FACTOR      1
+#define OAL_ROLLOFF_FACTOR      0.5f
 #define OAL_SPEED_OF_SOUND      343.3f
-// 128 map units per 3 meters (https://doomwiki.org/wiki/Map_unit).
-#define OAL_MAP_UNITS_PER_METER (128.0f / 3.0f)
+// 16 mu/ft (https://www.doomworld.com/idgames/docs/editing/metrics)
```

Comment:
So, we came from `42.6` map units per meter and are now at `52.4` map units per meter. Does this make any audible difference?

Reply:
I don't notice any difference but now it's consistent with the speedometer widget.